.palantir/revapi.yml

```diff
@@ -178,6 +178,12 @@ acceptedBreaks:
       old: "method void com.palantir.dialogue.annotations.StdDeserializer<T>::<init>(java.lang.String)"
       new: "method void com.palantir.dialogue.annotations.StdDeserializer<T>::<init>(java.lang.String)"
       justification: "no functional difference"
+  "1.128.0":
+    com.palantir.dialogue:dialogue-target:
+    - code: "java.class.kindChanged"
+      old: "interface com.palantir.dialogue.Response"
+      new: "class com.palantir.dialogue.Response"
```

Comment:
Per our discussion on the last PR, this is a temporary hack that will be reverted before the feature can be merged